integration_tests/environment.py

```diff
@@ -1,18 +1,20 @@
 from os import system
+import os
 
 
 def before_all(context):
     system(f'docker stop $(docker ps -a -q) && docker rm $(docker ps -a -q)')
     system(f'docker network create integtestnet')
     system('bash integration_tests/build_test_containers.sh')
+    os.environ["API_CLIENT_RUN_ON_REDIS"] = "true"
```

Comment:
It has already been set under `tox`

Reply:
yeah, I had some trouble with that. I wanted to run the integrationtests locally without `tox`. 
This line above didnt have any impact (only setting it directly in the shell with `export API_CLIENT_RUN_ON_REDIS="true"` worked.
Will remove the line above.